quantumdb-postgresql/src/test/java/io/quantumdb/core/backends/integration/videostores/AddColumnToCustomersTable.java

```diff
@@ -103,7 +105,11 @@ public void verifyTableStructure() {
 				.addColumn(new Column("customer_id", integer(), NOT_NULL))
 				.addColumn(new Column("rental_id", integer(), NOT_NULL))
 				.addColumn(new Column("date", date(), NOT_NULL))
-				.addColumn(new Column("amount", floats(), NOT_NULL));
+				.addColumn(new Column("amount", floats(), NOT_NULL))
+				.addColumn(new Column("amount_numeric", numeric(), NOT_NULL))
+				.addColumn(new Column("amount_numeric_precision", numeric(10), NOT_NULL))
+				.addColumn(new Column("amount_numeric_precision_scale", numeric(10,2), NOT_NULL))
+				.addColumn(new Column("invoice_pdf", bytea(), NOT_NULL));
```

Comment:
Maybe it would be best to make a `PostgresTypesTest` class instead to verify all of these can be parsed correctly. The videostores integration test suite was meant for debugging some more complex migrations (and avoid future regressions). They weren't really meant to test database types.